### ISSUE TITLE
Failed to parse source map error when using with source-map-loader

### ISSUE THREAD

**Issue by utgwkk** [susisu/promise-utils#11]
Using `@susisu/promise-utils` with source-map-loader yields a warning (see below) when building.
I tested it with [utgwkk/202204017-sketch-webpack-sourcemap](https://github.com/utgwkk/202204017-sketch-webpack-sourcemap) repository.

# Library version

- @susisu/promise-utils: 0.1.1
- webpack: 5.72.0
- source-map-loader: 3.0.1

# Output of build

```
% ENABLE_SOURCEMAP=true yarn build
yarn run v1.22.10
$ webpack
assets by status 403 bytes [cached] 1 asset
./src/index.js 225 bytes [built] [code generated]
./node_modules/@susisu/promise-utils/lib/index.js 1.07 KiB [built] [code generated] [1 warning]

WARNING in ./node_modules/@susisu/promise-utils/lib/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/utgwkk/ghq/github.com/utgwkk/202204017-sketch-webpack-sourcemap/node_modules/@susisu/promise-utils/src/index.ts' file: Error: ENOENT: no such file or directory, open '/home/utgwkk/ghq/github.com/utgwkk/202204017-sketch-webpack-sourcemap/node_modules/@susisu/promise-utils/src/index.ts'
 @ ./src/index.js 2:0-48 4:35-42

1 warning has detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.72.0 compiled with 1 warning in 285 ms
Done in 0.79s.
```

# Directory structure in `node_modules`

It looks like `src/index.ts` is not found.

```
% tree ./node_modules/@susisu/promise-utils
./node_modules/@susisu/promise-utils
├── LICENSE
├── README.md
├── lib
│   ├── index.d.ts
│   ├── index.d.ts.map
│   ├── index.js
│   └── index.js.map
└── package.json

1 directory, 7 files
```

**Comment by susisu** [comment]
@utgwkk Thanks for reporting! The repro really helped.

I have just published @susisu/promise-utils@0.1.2 which includes `src/index.ts` in the distribution package.
Can you try this out?

Hmm... Maybe all of my libraries are poor.
https://zenn.dev/suin/scraps/9f454023cefb67

**Comment by utgwkk** [comment]
Upgrading to v0.1.2 solved the issue! Thank you!

```
% yarn add @susisu/promise-utils
yarn add v1.22.10
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ @susisu/promise-utils@0.1.2
info All dependencies
└─ @susisu/promise-utils@0.1.2
Done in 6.11s.

% env ENABLE_SOURCEMAP=true yarn build
yarn run v1.22.10
$ webpack
asset main.70ba59efcfb368af548e.js 403 bytes [emitted] [immutable] [minimized] (name: main) 1 related asset
./src/index.js 225 bytes [built] [code generated]
./node_modules/@susisu/promise-utils/lib/index.js 1.07 KiB [built] [code generated]
webpack 5.72.0 compiled successfully in 439 ms
Done in 1.02s.
```
